components/atom/toast/src/config.js

```diff
@@ -0,0 +1,33 @@
+export const BASE_CLASS = 'react-AtomToast'
+
+export const POSITIONS = {
+  bottom: 'bottom',
```

Comment:
Since they are constants, what do you think about changing them to constants using snake_case? Or is this a `sui` convention?